test/fixtures/us_core_examples/Patient-ac1bdb14-fea1-4912-8d7c-e3ecec74b0d7.json

```diff
@@ -0,0 +1,144 @@
+       {
+          "system":"https://github.com/synthetichealth/synthea",
+          "value":"ac1bdb14-fea1-4912-8d7c-e3ecec74b0d7"
+       },
+       {
+          "type":{
+             "coding":[
+                {
+                   "system":"http://terminology.hl7.org/CodeSystem/v2-0203",
+                   "code":"MR",
+                   "display":"Medical Record Number"
+                }
+             ],
+             "text":"Medical Record Number"
+          },
+          "system":"http://hospital.smarthealthit.org",
+          "value":"ac1bdb14-fea1-4912-8d7c-e3ecec74b0d7"
+       },
+       {
+          "type":{
+             "coding":[
+                {
+                   "system":"http://terminology.hl7.org/CodeSystem/v2-0203",
+                   "code":"SS",
+                   "display":"Social Security Number"
+                }
+             ],
+             "text":"Social Security Number"
+          },
+          "system":"http://hl7.org/fhir/sid/us-ssn",
+          "value":"999-53-4901"
+       },
+       {
+          "type":{
+             "coding":[
+                {
+                   "system":"http://terminology.hl7.org/CodeSystem/v2-0203",
+                   "code":"DL",
+                   "display":"Driver's License"
+                }
+             ],
+             "text":"Driver's License"
+          },
+          "system":"urn:oid:2.16.840.1.113883.4.3.25",
+          "value":"S99943125"
+       }
+    ],
+    "name":[
+       {
+          "use":"official",
+          "family":"Becker968",
+          "given":[
+             "Sadie426"
+          ],
+          "prefix":[
+             "Mrs."
+          ]
+       },
+       {
+          "use":"maiden",
+          "family":"Harber290",
+          "given":[
+             "Sadie426"
+          ],
+          "prefix":[
+             "Mrs."
+          ]
+       }
+    ],
+    "telecom":[
+       {
+          "system":"phone",
+          "value":"555-347-7769",
+          "use":"home"
+       }
+    ],
+    "gender":"female",
+    "birthDate":"1948-01-24",
+    "address":[
+       {
+          "extension":[
+             {
+                "extension":[
+                   {
+                      "url":"latitude",
+                      "valueDecimal":42.315016452625876
+                   },
+                   {
+                      "url":"longitude",
+                      "valueDecimal":-71.06982421556008
+                   }
+                ],
+                "url":"http://hl7.org/fhir/StructureDefinition/geolocation"
+             }
+          ],
+          "line":[
+             "1047 Bartoletti Manor Unit 79"
+          ],
+          "city":"Boston",
+          "state":"MA",
+          "postalCode":"02113",
+          "country":"US"
+       }
+    ],
+    "maritalStatus":{
+       "coding":[
+          {
+             "system":"http://terminology.hl7.org/CodeSystem/v3-MaritalStatus",
+             "code":"M",
+             "display":"M"
```

Comment:
I am going to delete this.  I pulled it from the SMART Bulk Data response because it triggered a warning, but turns out that warning isn't triggered in 4.0.1 apparently.  So it doesn't actually doing anything additionally useful any more.